rosdep/python.yaml

```diff
@@ -5814,6 +5826,26 @@ python3-setuptools:
   openembedded: [python3-setuptools@openembedded-core]
   rhel: ['python%{python3_pkgversion}-setuptools']
   ubuntu: [python3-setuptools]
+python3-sexpdata:
+  debian:
+    '*': [python3-sexpdata]
+    jessie:
+      pip:
+        packages: [sexpdata]
+    stretch:
+      pip:
+        packages: [sexpdata]
+  ubuntu:
+    '*': [python3-sexpdata]
+    bionic:
+      pip:
+        packages: [sexpdata]
+    trusty:
+      pip:
+        packages: [sexpdata]
```

Comment:
Trusty is out of support, so we don't need to add support for it.  Just remove this.